src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-reweight-modal/osd-reweight-modal.component.html

```diff
@@ -0,0 +1,47 @@
+<cd-modal [modalRef]="bsModalRef">
+  <ng-container class="modal-title"
+                i18n>
+    Reweight OSD
+  </ng-container>
+
+  <ng-container class="modal-content">
+    <form class="form-horizontal"
+          [formGroup]="reweightForm">
+      <div class="modal-body">
```

Comment:
The `has-error` class is missing here to color the error.

```suggestion
      <div class="modal-body" [ngClass]="{'has-error': weight.errors}">
```